core/example/solvers/pme.hpp

```diff
@@ -0,0 +1,446 @@
+#include "definitions.h"
```

Comment:
What is the difference between this and whats in `spme`?

Reply:
There is no difference. I was certain that I had removed this, but I guess it was missed. 
`main.cpp` should also be gone - I took that and singled out each example (`directsum_example.cpp`, `ewaldsum_example.cpp`, and `spme_example.cpp`)